concent-builder/configure.yml

```diff
@@ -111,3 +111,10 @@
             state: directory
             owner: "{{ shared_user }}"
             group: "{{ shared_user }}"
+
+        - name: Upload configuration for kubectl
+          copy:
+            src:   "{{ local_secret_dir }}/cloud/config"
+            dest:  "/home/{{ shared_user }}/.kube/config"
```

Comment:
Replacing the whole `~/.kube/config` has basically the same effect as `gcloud container clusters get-credentials`. It still replaces the configuration globally and does not let us manage multiple clusters in parallel which was one of the problems listed in the issue.

We need a better solution. Is it possible to have several configurations and specify them by path in `kubectl`? Or maybe we could install the configurations in locations like `~/.kube/config.d/concent-dev`, `~/.kube/config.d/concent-staging`, etc and then add a step that generates `~/.kube/config` by merging them?

Reply:
We don't need a few configuration file. Kubectl allow to specify context by `--context=name` like for example `concent-dev`. This solution allow manage multiple clusters in parallel.

Reply:
After our discussion I understand that this file contains credentials for all the clusters. If we treat it as a part of the cloud secrets then it's fine. I mistook it for a part of cluster secrets that would need to change for each cluster.